codesigndoc/codesigngroup.go

```diff
@@ -184,7 +184,8 @@ func collectIpaExportCodeSignGroups(archive xcarchive.IosArchive, installedCerti
 
 		if len(filteredCodeSignGroups) == 0 {
 			fmt.Println()
-			log.Errorf(collectCodesigningFilesInfo)
+			log.Errorf("🚨  Could not find any valid code signing file for the selected export method:")
```

Comment:
`Could not find the codesigning files for {app-store|ad-hoc|development|enterprise} ipa export:`